vpr/src/draw/draw_basic.h

```diff
@@ -69,9 +69,29 @@ void drawroute(enum e_draw_net_type draw_net_type, ezgl::renderer* g);
 void draw_routed_net(ParentNetId net, ezgl::renderer* g);
 
 //Draws the set of rr_nodes specified, using the colors set in draw_state
+//draw_partial_route() takes care of layer visibility and cross-layer settings
 void draw_partial_route(const std::vector<int>& rr_nodes_to_draw,
                         ezgl::renderer* g);
 
+
+/**
+ * @brief Returns the layer number of a timing path node
+ * @param node
+ * @return layer number the node is situated on.
+ */
+int get_timing_path_node_layer_num(tatum::NodeId node);
+
+/**
+ * @brief Draws if both the current_node and prev_node are on the same layer and it is visible,
```

Comment:
Returns true if ... (it doesn't actually do the drawing so best not to say it draws it).

Otherwise returns false.